### DISCUSSION TITLE
How WSA (together with WSA PacMan) treats apps' access to hard disk, contacts, camera, phone call history, SMS messages, some of which do not even exist on Windows?

### DISCUSSION THREAD

**Discussion by victorxstc** [MustardChef/WSABuilds#359] [Q&A]
Thanks for your great software. I am very new to WSA and WSA-PacMan and am not oriented. My questions has a technical part and a security part:

**Security part:** Is WSA something like an emulator? I installed WSA **and WSA PacMan**. So I don't know if I install an Android app that is a secret (unknown) spyware, what data is exposed to the app by Windows? For example, is my computer's hard disk 'visible' to the spyware? What about camera or mic? Is anything on Windows visible to the Android app? etc. 

For example, in BlueStacks, I know that if I install a spyware, it is contained within the emulator sandbox and would not have access to anything outside BlueStacks. What about WSA? Does it work like a sandbox too? **I should clarify that** I have also installed WSA **PacMan** that allows Windows to treat APK files like EXE files. So it worries me that perhaps in the same way, it might allow Windows and all information on hard disks to be visible to Android apps.

**Technical part:** Besides, some app permissions do not make sense on Windows. For example, Windows does not have contact lists, phone call history, SMS, etc. How WSA treat them? Will there be problems? 

Thanks a lot.

**Comment by alesimula** [comment]
@victorxstc Hello, I'm the developer of WSA PacMan and somehow found this discussion by complete chance... that said, WSA PacMan is just a graphical package installer for WSA (makes installing .apk easier), and WSA (Windows Subsystem for Android) is a tool developed by Microsoft that allows to run Android applications on Windows (sadly discontinued by Microsoft)

You can see it as some kind of Android emulator, and the file access the applications have is limited to the emulated android system, do not have access to any other data on your PC
